Tests/CryptoTests/Signatures/EdDSA/EdDSATests.swift

```diff
@@ -75,4 +75,54 @@ class EdDSATests: XCTestCase {
         // This signature should be invalid
         XCTAssertFalse(privateKey.publicKey.isValidSignature(DispatchData.empty, for: DispatchData.empty))
     }
+	
+	func testCurve25519SigningPublicKeyEquatable() throws {
```

Comment:
Part of #173 but I was unable to create a diff between just this PR and #173 since my branch is in a fork, otherwise I would have created this PR with the branch `curve25519_publickeys_equatable` as the target branch, so that we could have seen only the relevant diff.